src/VirtoCommerce.Platform.Assets.AzureBlobStorage/AzureBlobProvider.cs

```diff
@@ -40,12 +40,13 @@ public virtual async Task<BlobInfo> GetBlobInfoAsync(string blobUrl)
             if (string.IsNullOrEmpty(blobUrl))
                 throw new ArgumentNullException(nameof(blobUrl));
 
-            var uri = blobUrl.IsAbsoluteUrl() ? new Uri(blobUrl) : new Uri(_cloudBlobClient.BaseUri, blobUrl.TrimStart('/'));
+            var uri = blobUrl.IsAbsoluteUrl() ? new Uri(blobUrl) : new Uri(_blobServiceClient.Uri, blobUrl.TrimStart(Delimiter[0]));
             BlobInfo retVal = null;
             try
             {
-                var cloudBlob = await _cloudBlobClient.GetBlobReferenceFromServerAsync(uri);
-                retVal = ConvertBlobToBlobInfo(cloudBlob);
+                var blob = new BlobClient(new Uri(_blobServiceClient.Uri, uri.AbsolutePath.TrimStart('/')));
```

Comment:
Since the storage account is based on transaction volume, it is better to handle the 404 (BlobNotFound) exception because that is just one api call, as opposed to checking the BlobClient.ExistsAsync() first and then making the BlobClient.DownloadAsync() call (2 api transactions).
Based on: https://elcamino.cloud/articles/2020-03-30-azure-storage-blobs-net-sdk-v12-upgrade-guide-and-tips.html